src/health-tm/health-tm-lambda.js

```diff
@@ -0,0 +1,82 @@
+const fetch = require('node-fetch')
+
+const TM_STATUS_URL =
+  'https://tasking-manager-tm4-production-api.hotosm.org/api/v2/system/heartbeat/'
```

Comment:
let's pass these as env variables in the template, so if this changes, the template configuration can be modified, and we won't have to touch the code. : ) Add some sanity checks here for the URL.